index.html

```diff
@@ -47,5 +53,235 @@ <h2>LET’S CONNECT</h2>
         </div>
       </div>
     </section>
+    <section class="main">
+      <section class="works">
+        <div class="card">
+          <div class="portfolio-snapshoot">
+            <img
+              src="assets/images/snapshoot-portfolio-1.png"
+              alt="portfolio-1"
+            />
+          </div>
+          <div class="left-block">
+            <div class="portfolio-info">
+              <p class="portfolio-title">TONIC</p>
```

Comment:
- [ ] As said by the previous reviewer, Kindly make sure your project title uses the header tag `hx`